src/com/zipcode/logan/aliceandbob/Person.java

```diff
@@ -0,0 +1,31 @@
+package com.zipcode.logan.aliceandbob;
+
+public class Person {
+
+    private String name;
+    private boolean acceptable;
+
+    public Person(String str) {
+        name = str;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String s) {
+        name = s;
+    }
+
+    public String capitalizeFirst() {
```

Comment:
`capitalizeFirst` is a procedural utility method.
This method should be extracted and placed in some `StringUtils` class.

```java
public class StringUtils {
    public static String capitalizeFirst(String s) {
        return s.substring(0,1).toUpperCase() + s.substring(1); 
    }
}
```